src/NUnitEngine/nunit.engine.tests/Extensibility/ExtensionAssemblyTests.cs

```diff
@@ -22,96 +22,60 @@
 // ***********************************************************************
 
 using System;
-using System.Collections.Generic;
 using System.Reflection;
-using System.Text;
+using NUnit.Engine.Extensibility;
 using NUnit.Framework;
 
```

Comment:
Significant changes in this file are mainly due to tests here split out into ExtensionSelectorTests